### DISCUSSION TITLE
post/cloudflare-worker-region/

### DISCUSSION THREAD

**Discussion by giscus** [lyc8503/lyc8503#22] [Announcements]
# post/cloudflare-worker-region/

UptimeFlare 是我之前写的一个开源项目, 用途类似 uptime-kuma 或 UptimeRobot, 用于监控网站/服务器的在线状态, 展示状态页. 区别是 UptimeFlare 是完全 serverless 的, 可以在 Cloudflare Workers 上部署. 同时, UptimeFlare 还有个自建方案无法比拟的优势: 它能利用 Cloudflare 在全球

https://blog.lyc8503.net/post/cloudflare-worker-region/

**Comment by miantiao-me** [comment]
使用 nmap 扫描不到 IP

是不是需要挂 warp 代理？

**Reply by lyc8503** [reply]
不需要挂 WARP, 如果扫不到的话可以多尝试几个段, 或者试试附近其他地区的.

**Reply by miantiao-me** [reply]
使用 sudo 后能扫到了。但是配置的 IP 会出现 522 错误。

当前使用的 IP 是 8.20.100.1

**Reply by miantiao-me** [reply]
我刚发现 xx.1 是不工作的，而我几个都使用了 .1

**Reply by lyc8503** [reply]
> 我刚发现 xx.1 是不工作的，而我几个都使用了 .1

是的，Wiki 中提到了 `.1` 不能使用

**Reply by miantiao-me** [reply]
感谢你的分享，我已经成功使用起来。 

而且我发现一个 CF 自己公开的当地 IP 段 https://api.cloudflare.com/local-ip-ranges.csv

我也对网络感兴趣，也有一个 ASN。方便的话加一下微信。 ID 和 GitHub 一致

**Reply by lyc8503** [reply]
> 而且我发现一个 CF 自己公开的当地 IP 段 https://api.cloudflare.com/local-ip-ranges.csv

这个看起来不错，下次更新可能使用这个，比在整个库里搜方便。

**Reply by gandli** [reply]
hi，博主，我在思考，利用 Cloudflare ASN 中所有位置的所有 IP https://api.cloudflare.com/local-ip-ranges.csv，让 cloudflare workers 代理  http 请求在指定区域执行，可行吗？

**Comment by rainlycoris** [comment]
请问怎么找日本的 ip 啊，扫了半天没找到，感谢

**Reply by rainlycoris** [reply]
脑残了，没仔细看上面的文件，打扰了

**Reply by rainlycoris** [reply]
唉但是扫出来没一个能用的

**Comment by suykjay** [comment]
2025/06，这个方法似乎用不了了？一直是1002报错。。。

**Reply by lyc8503** [reply]
观察到有些账号开始不能使用了，但另一些账号一直可以。

不过 CF 给出了官方支持的选择地区方法：https://github.com/lyc8503/UptimeFlare/wiki/Check-proxy-setup

**Comment by marcobibichan** [comment]
那可不可以一個域名用不同cname指定不同地區的8.x.x.x的ip?

**Reply by lyc8503** [reply]
理论可行，不过这种方法现在疑似已经似了，很多账号不能使用。
